common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
You should go with the second option.  However, to clarify the first option, I meant that if you wanted to use `HTML(msg)` in the template, this would keep it from escaping because you are declaring that it is already HTML.  For plain text, you would have to escape it early and turn it into HTML if you are declaring it as such.

I think you should go with the second option. I was just trying to clarify how the former code would have to work to keep things safe.